nginx/Dockerfile

```diff
@@ -18,10 +18,10 @@ ARG SIG_ALG="dilithium3"
 ARG DEFAULT_GROUPS=x25519:x448:kyber512:p256_kyber512:kyber768:p384_kyber768:kyber1024:p521_kyber1024
 
 # define the nginx version to include
-ARG NGINX_VERSION=1.23.3
+ARG NGINX_VERSION=1.25.2
 
 # Define the degree of parallelism when building the image; leave the number away only if you know what you are doing
-ARG MAKE_DEFINES="-j 18"
+ARG MAKE_DEFINES="-j"
```

Comment:
This change will make any "few cores" machine running this squeak, possibly crash (have seen this before). I appreciate that "modern laptops" couldn't care less but we don't prescribe this for building the demos. Also, some CI systems (notably CCI) has problems with unlimited "-j". The 18 was what CCI could handle without failures. Unless you can point to a document stating improved CCI capabilities, please revert.

Reply:
Sorry I didn't intend to make the machines and CI squeak :) Reverted in 25ea3f839e987388c2346b24e806a8a4adcf5bd2